src/mail/view/MailView.ts

```diff
@@ -764,7 +764,7 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 	 */
 	private renderHeaderView(): Children {
 		const selectedEntities = this.cache.mailList?.list.getSelectedEntities() ?? []
-		const selectNone = () => this.cache.mailList?.list.selectNone ?? noOp
+		const selectNone = (() => this.cache.mailList?.list.selectNone()) ?? noOp
```

Comment:
what the hell is going on here 😅

either the whole function would be nullable (why?) or the result would be (why do we pass no-op then?). Right now this `??` does nothing because it's left side is an arrow function